### PR TITLE
Add option to specify routes by hand

### PR DESCRIPTION
**Pull request by mika-fischer**
Format is { route: '/foo/{id}', handlers: require('./foo') }

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -671,6 +671,9 @@ post.apiDoc = {
 
 ```
 
+Alternatively, args.paths may contain route specifications of the form
```
Comment (jsdevel) [inline README.md]: I realize we'll end up internally merging these with `fs-routes`; however, on second look we should probably stay more inline with openapi concepts.  How about something like this? `{path: '/foo/{id}', module: require('./handlers/foo')}`

Comment (mika-fischer) [inline README.md]: OK, fine with me, I'll update the pull request accordingly.

Comment (mika-fischer) [inline README.md]: Done, have a look

Comment (jsdevel) [inline README.md]: Looks great!  Thank you so much for adding this!  I feel it gets us closer to resolving #12.

**Comment by jsdevel** [comment]
Published as `v0.31.0` @mika-fischer

**Pull request merged**